bigquery/google/cloud/bigquery/job.py

```diff
@@ -506,6 +529,11 @@ def output_rows(self):
     https://cloud.google.com/bigquery/docs/reference/rest/v2/jobs#configuration.load.allowQuotedNewlines
     """
 
+    autodetect_schema = AutoDetectSchema('autodetect_schema', bool)
```

Comment:
Hi @tswast, thanks for the review :)! Sure, I can update that (I called it as it is thinking on being explicit for the end user that this property is related to detection of schemas, but agree that it changes overall structure of how the code has been built so far).

Just wondering, how should I proceed on updating the implementations of your reviews? Should I just commit the changes and make the PR again? Is there some required steps you recommend to follow?

Thanks!